src/main/java/bio/terra/service/snapshot/SnapshotSummary.java

```diff
@@ -161,4 +161,12 @@ public SnapshotSummaryModel toModel() {
   private List<StorageResourceModel> toStorageResourceModel() {
     return getStorage().stream().map(StorageResource::toModel).collect(Collectors.toList());
   }
+
+  /**
+   * @param model a `SnapshotSummaryModel`
+   * @return whether the underlying snapshot can be accessed via RAS passport authorization
+   */
+  public static boolean passportAuthorizationAvailable(SnapshotSummaryModel model) {
```

Comment:
@tlangs please let me know if this addresses your request.

Because the `SnapshotSummaryModel` is a generated class, we can't modify it directly.  If you think that the determination of a snapshot's passport eligibility should live somewhere other than a `SnapshotSummary` class method, let's chat.

Reply:
I think there's a bigger issue here in that `SnapshotSummaryModel` contains more information than `Snapshot`. I think there's an opportunity here for more substantive changes, but agree that that sounds a lot like scope creep.

Reply:
I agree with you, the delineation between what is a good fit for the summary vs. what might be extracted from the snapshot directly is unclear to me.  I'd be interested in thinking more about this outside of this PR!